pkg/cloud/services/compute/instance.go

```diff
@@ -562,8 +568,18 @@ func applyServerGroupID(opts servers.CreateOptsBuilder, serverGroupID string) se
 	return opts
 }
 
-// Helper function for getting image id from name.
-func (s *Service) getImageIDFromName(imageName string) (string, error) {
+// Helper function for getting image ID from name or ID.
```

Comment:
>>>Helper function for getting image ID from name or ID.

this seems confusing

Reply:
I agree :smile: However, despite what GitHub says I didn't write this. This is being picked up as a moved line when in fact I simply renamed the function from `getImageID` to `GetImageID` and folded `getImageIDFromName` into this. It's also technically true.